src/main/kotlin/avans/avd/incidents/IncidentRoute.kt

```diff
@@ -0,0 +1,125 @@
+package avans.avd.incidents
+
+import avans.avd.exceptions.MissingRoleException
+import avans.avd.users.Role
+import avans.avd.utils.*
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.auth.*
+import io.ktor.server.plugins.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+
+fun Route.incidentRoute(
+    incidentService: IncidentService
+) {
```

Comment:
It seems git failed to create a decent diff with the routing files...

One important change here is that, instead of using the route plugin for checking the roles, I just introduced an extension method `assertHasRole()`.  This way, we remove a layer of nesting, and all the `authenticate` endpoints can be grouped together.